patterns/patterns.toml

```diff
@@ -101,3 +101,18 @@ name = "heartbeat"
 script = "heartbeat.py"
 program = "python3"
 options = { color_map_name = "Reds"}
+
+[[patterns]]
+name = "mako"
+script = "mako.js"
+program = "node"
+
+[[patterns]]
+name = "swirl"
```

Comment:
I'm going to opt to remove this one because in going through all of the patterns, this one is pretty similar to faces_linear